migrations/00000000000000-init.js

```diff
@@ -1,14 +1,15 @@
 module.exports = {
   async up(db) {
     // Create indexes for the 'jobs' collection
-    await db
-      .collection("jobs")
-      .createIndexes([
-        { key: { id: 1 } },
-        { key: { job_type: 1, internal_id: -1 }, unique: true },
-        { key: { job_type: 1, status: 1, internal_id: -1 } },
-        { key: { status: 1 } },
-      ]);
+    await db.collection("jobs").createIndexes([
+      { key: { id: 1 } },
+      { key: { job_type: 1, internal_id: -1 }, unique: true },
+      { key: { job_type: 1, status: 1, internal_id: -1 } },
+      { key: { status: 1 } },
+      // primarily for get_jobs_without_successor
+      { key: { job_type: 1, status: 1 } },
+      { key: { job_type: 1, internal_id: 1 } },
```

Comment:
we already have `{ job_type: 1, internal_id: -1 }`, are you sure we need this?

Reply:
Yup, initially I tried with only ` { key: { job_type: 1, status: 1 } }` and the query time actually increased from average 51sec to average 58sec and only after adding `{key: { job_type: 1, internal_id: 1 } }` it reduced to `340ms`